runtime.txt

```diff
@@ -1 +1 @@
-python-3.8.8
+python-3.9.6
```

Comment:
I'm not convinced that Cloudfoundry's Python buildpack has 3.9.6 yet. I see https://github.com/cloudfoundry/python-buildpack/releases that lists up through 3.9.5, and I'm not totally sure how to make sure that cloud.gov is running the latest 1.7.42 release of that Python buildpack.

Can you try deploying this to `tock-staging` using the process in https://github.com/18F/tock/blob/main/docs/deployment-process.md and see if it is able to find Python 3.9.6 and get up and running?

Reply:
I will do that. It looks like there's also an issue with finding 3.9.6 on circleci, so I will look into that as well.

Reply:
I was able to verify that cloud.gov's buildpack does not support 3.9.6 by pushing to my personal sandbox. Looks like 3.9.5 is the most recently supported version. I updated the title & description to reflect this.

Reply:
3.9.6 is now listed there. Do we want to stay with 3.9.5 or go ahead and do 3.9.6? @matthinz @neilmb

Reply:
I don't care either way. Take this PR as-is and do another to upgrade to 3.9.6? First figure out a policy on when and how we make these changes?

Reply:
RE: Policy, it looks like there are two sources of Python version updates:

1. cloud.gov buildpack changes
2. Snyk security updates

It seems like cloud.gov changes will likely require an engineer periodically reviewing the state of staging/prod and updating. Snyk seems like it's good at poking us when new versions come out. I'm happy as long as the local dev environment stays in sync. I tried to address this process in [the docs](https://github.com/18F/tock/pull/1299/files#diff-3454e22d3d03c40d515cf0b06e7436f912886f8ae838228b6522e5454f6e518aR37)

Reply:
I got email from cloud.gov that they've updated their buildpacks today, so there may be additional versions available. (3.9.6)

Reply:
https://github.com/cloudfoundry/python-buildpack/releases

looks like 3.9.6 is now included...